libpod/events/config.go

```diff
@@ -59,6 +59,8 @@ type Details struct {
 	// Attributes can be used to describe specifics about the event
 	// in the case of a container event, labels for example
 	Attributes map[string]string
+	// Whether state change happened for HealthStatus or not
+	IsHealthStatusChanged bool
```

Comment:
This needs to be at least omitempty like other fields so it doesn't show up on non hc events. Maybe it would be better to add this into into the Attributes map instead, @mheon WDYT?

Reply:
By reading the code, I understand that attribute map is fully allocated to storing container's labels 
That is why I decided not to choose that. ( Initially I thought to store this in attributes map )

Reply:
@Luap99 , @mheon : Please let me know what is the correct place to store these data. I will rebase, make changes & test my changes accordingly